Client/Frontend/Fakespot/FakespotCoordinator.swift

```diff
@@ -0,0 +1,32 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+protocol FakespotCoordinatorDelegate: AnyObject {
+    // Define any coordinator delegate methods here
```

Comment:
I think we should have a `didFinish` or some equivalent in there? Otherwise the coordinator won't deinit and we'll stay part of the browser coordinator as child

Reply:
I will keep this empty for now because I'm not sure if I need it, but parentCoordinator will be ParentCoordinatorDelegate and I will call didFinish from there. Coming up with a PR update.